naturebridge.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+---
+### Venue information
+
+2019 Ruby by the Bay will be held March 28 - April 1 (Friday - Monday) at NatureBridge in Sausalito, California.
+
+![Aerial photo of venue next to the ocean](/images/naturebridge_from_above.jpg "Aerial photo of venue next to the ocean"){:class="img-responsive"}
+
+###### What is this beautiful place??
+
+![Photo of NatureBridge signage in front of some of their buildings](/images/naturebridge_sign.jpg "Photo of NatureBridge signage in front of some of their buildings"){:class="img-responsive"}
+
+NatureBridge is a nonprofit that focuses on educating kids about the environment.  They have school groups during the school year, and run day camps during the summers.
+
+###### Seriously how is it this beautiful??
+
+NatureBridge is lucky enough to lease their space from the National Park Service. It's housed in Golden Gate National Recreation Area. The campus was originally created by the Army.
+
+###### What will sleeping arrangements be like?
+We have four bunkhouses:
+* Non-binary people (Owl's Roost)
+* Women's (Heron)
+* Men's (Pelican)
+* All genders (Raven)
+
```

Comment:
Maybe we actually don't need to get specific and name the dorms, since we don't know the breakdown of who will attend and their preferences it might be hard to allocate dorms and if we have this here we're making a commitment super early to people where they will be staying.